plugins/sensors/upsmonpro_

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/perl
+
+=head1 NAME
+
+upsmonpro_ - Munin plugin to monitor Powercom UPS via UPSMON PRO program L<http://www.pcm.ru/support/soft/>
+
+=head1 INSTALLATION
+
+  /etc/munin/plugins/upsmonpro_load -> /usr/share/munin/plugins/upsmonpro_
+  /etc/munin/plugins/upsmonpro_status -> /usr/share/munin/plugins/upsmonpro_
+  /etc/munin/plugins/upsmonpro_temp -> /usr/share/munin/plugins/upsmonpro_
+  /etc/munin/plugins/upsmonpro_voltage -> /usr/share/munin/plugins/upsmonpro_
+
+=head1 CONFIGURATION
+
+Environment variables:
+
+  host - UPSMON PRO server host, default localhost
+  port - UPSMON PRO port, default 2601
+
+=head1 MAGIC MARKERS
+
+  #%# family=auto
+  #%# capabilities=autoconf
```

Comment:
Since you are offering multiple modes (temp, ...), it would be great if you could add support for the [suggest capability](http://munin-monitoring.org/wiki/PluginConcise#suggest).